### PR TITLE
settings cleanup

### PR DESCRIPTION
**Pull request by Vendicated**
Uses data attributes to have less ugly js

### CODE REVIEW

**Comment by Vendicated** [comment]
btw lang logic seems borked cause the file is named "fr_FR" but this is what's in the lang.json 
image

**Comment by Vendicated** [comment]
Btw it would be nice to remove all the obsolete strings from the html once string loading works correctly and every string is covered

**Comment by Vendicated** [comment]
this is basically ready now. Do you want me to move the rest of the strings that aren't in the lang file over and remove all the strings from the html?

**Comment by smartfrigde** [comment]
Yes and yes if possible

**Comment by Vendicated** [comment]
Lol i forgot change listeners

**Comment by Vendicated** [comment]
wait nvm there aren't any listeners, but the switches are broken

File: src/settings/settings.html [hunk 1]
```diff
@@ -9,264 +9,218 @@
...
         <div class="switch acWebsocket">
-            <label class="header" id="settings-invitewebsocket">Rich Presence (Experimental)</label>
-            <input class="tgl tgl-light left" id="websocket" type="checkbox" />
+            <label class="header" data-string="settings-invitewebsocket"></label>
+            <input id="websocket" class="tgl tgl-light left" data-setting="inviteWebsocket" type="checkbox" />
             <label class="tgl-btn left" for="websocket"></label>
             <p class="description">
+                <!-- TODO add data-string -->
                 Uses <a target="_blank" href="https://github.com/OpenAsar/arrpc">arRPC</a> to support Discord RPC (Rich
                 Presence) with local programs on your machine. Work in progress.
```
Comment (Vendicated) [inline src/settings/settings.html:70]: this is the only string left now

**Comment by Vendicated** [comment]
I just realised there's a second settings file hummus.html. the only difference i can tell is in the client mods. Could just merge that into the main file with some js?

File: src/settings/settings.html (hunk 1 above)
Comment (smartfrigde) [reply src/settings/settings.html:70]: It's fine

**Pull request merged**